redash/models.py

```diff
@@ -315,29 +204,30 @@ def is_public(self):
 
     @property
     def admin_group(self):
-        return self.groups.where(Group.name=='admin', Group.type==Group.BUILTIN_GROUP).first()
+        return self.groups.filter(Group.name == 'admin', Group.type == Group.BUILTIN_GROUP).first()
 
     def has_user(self, email):
-        return self.users.where(User.email==email).count() == 1
+        return self.users.filter(User.email == email).count() == 1
 
 
-class Group(BaseModel, BelongsToOrgMixin):
+class Group(db.Model, BelongsToOrgMixin):
     DEFAULT_PERMISSIONS = ['create_dashboard', 'create_query', 'edit_dashboard', 'edit_query',
                            'view_query', 'view_source', 'execute_query', 'list_users', 'schedule_query',
                            'list_dashboards', 'list_alerts', 'list_data_sources']
 
     BUILTIN_GROUP = 'builtin'
     REGULAR_GROUP = 'regular'
 
-    id = peewee.PrimaryKeyField()
-    org = peewee.ForeignKeyField(Organization, related_name="groups")
-    type = peewee.CharField(default=REGULAR_GROUP)
-    name = peewee.CharField(max_length=100)
-    permissions = ArrayField(peewee.CharField, default=DEFAULT_PERMISSIONS)
-    created_at = DateTimeTZField(default=datetime.datetime.now)
+    id = Column(db.Integer, primary_key=True)
+    org_id = Column(db.Integer, db.ForeignKey('organizations.id'))
+    org = db.relationship(Organization, backref="groups")
+    type = Column(db.String(255), default=REGULAR_GROUP)
+    name = Column(db.String(100))
+    permissions = Column(postgresql.ARRAY(db.String(255)),
+                         default=DEFAULT_PERMISSIONS)
+    created_at = Column(db.DateTime(True), default=db.func.now())
```

Comment:
Probably, but I don't want to change the schema until after I get things working as-is. (Hi Jeff! It's been a while! Never expected to see someone from SFSH commenting on my code :-)